.flake8

```diff
@@ -3,6 +3,8 @@ max-line-length = 120
 exclude=
     demo.py
     docs/conf.py
+ignore =
+    E731
```

Comment:
Don't love global ignores. [This rule](https://www.flake8rules.com/rules/E731.html) does makes sense to me as well despite the worse code style around lambdas.